security/pkg/nodeagent/secretfetcher/secretfetcher.go

```diff
@@ -505,3 +505,7 @@ func (sf *SecretFetcher) AddSecret(obj interface{}) {
 func (sf *SecretFetcher) DeleteSecret(obj interface{}) {
 	sf.scrtDeleted(obj)
 }
+
+func (sf *SecretFetcher) ResetClientConnectionForCertRotation(isRotate bool) error {
+	return sf.CaClient.Reconnect(isRotate)
```

Comment:
this is just a one line function, can we just call sf.CaClient.Reconnect() directly?

Reply:
yes will refactor that